server/controllers/customer.js

```diff
@@ -0,0 +1,88 @@
+const stripe = require("stripe")(process.env.STRIPE_SK);
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+
+exports.createCustomer = asyncHandler(async (req, res) => {
+  const { billing_details, id } = req.body;
+  const { userId } = req.params;
+  if (!userId || !id) {
+    return res
+      .status(404)
+      .json({ err: "userId or paymentMethodId is undefined" });
+  }
+  try {
+    const customer = await stripe.customers.create(billing_details);
```

Comment:
Suggestion: It would be nice to check if a user already has a `stripeId` and only create a customer for their account in case they don't have the `stripeId`